OCaml/OCaml.sublime-syntax

```diff
@@ -505,7 +505,7 @@ contexts:
         - match: (?=\b(type|val|external|class|module|end)\b)|^\s*$
           pop: true
         - match: '(\?)([a-z][a-zA-Z0-9_]*)\s*(:)'
-          scope: variable.parameter.ameter.optional.ocaml
+          scope: variable.parameter.optional.ocaml
```

Comment:
This was odd... I'm hoping it was just a copy of the previous and can be elided.